CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
I found that `stdlib/public/core/CMakeLists.txt` uses `if(CMAKE_SYSTEM_NAME STREQUAL ...` comparing with "Darwin" and "FreeBSD". To be turned on, we should scan all CMakeLists.txt and *.cmake files and test build for all platforms. That was not simple job to me.